packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -448,63 +448,63 @@ export const guides = [
         title: "Authorization",
         subLinks: [
           {
-            title: "Implement authorization by grant type",
+            title: "Implement authz by grant type",
             guideName: "implement-grant-type",
           },
           {
-            title: "Create an Authorization Server",
+            title: "Create an authorization server",
             guideName: "customize-authz-server",
           },
           {
-            title: "Configure SSO for Native apps",
+            title: "Configure SSO for native apps",
             guideName: "configure-native-sso",
           },
           { title: "Request user consent", guideName: "request-user-consent" },
           {
-            title: "Configure Device Authorization Grant Flow",
+            title: "Configure the Device Authz grant flow",
             guideName: "device-authorization-grant",
           },
           {
-            title: "Client secret rotation and key management",
+            title: "Secret rotation and key management",
             guideName: "client-secret-rotation-key",
           },
           {
             title: "Tokens",
             subLinks: [
               {
-                title: "Build a JWT for Client Authentication",
+                title: "Build a JWT for client authn",
                 guideName: "build-self-signed-jwt",
               },
               {
-                title: "Customize tokens returned from Okta with custom claims",
+                title: "Customize tokens: Custom claims",
```

Comment:
It seems odd having a bunch of "Customize tokens:" prefixes instead of a subcategory. How about either adding a subcategory or replacing "Customize tokens:" with "Add"... It's easier to read and easier on the eye.
```suggestion
                title: "Add custom claims",
```

Reply:
I didn't want to create a sub-sub category - so I went the other route and I think it works. Thanks!